endpoints-support.md

```diff
@@ -189,34 +189,36 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
-|<sub>/rest/logical-interconnects/{id}/unassignedUplinkPortsForPortMonitor</sub>          | GET       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/logical-interconnects/{id}/configuration</sub>                                | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/logical-interconnects/{id}/port-monitor</sub>                                 | GET       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/logical-interconnects/{id}/port-monitor</sub>                                 | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/logical-interconnects/{id}/telemetry-configurations/{tcId}</sub>              | GET       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/logical-interconnects/{id}/telemetry-configurations/{tcId}</sub>              | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/logical-interconnects/compliance</sub>                                        | PUT      | :heavy_multiplication_x:       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |
-|<sub>/rest/logical-interconnects/{id}</sub>                                              | PATCH     | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   |
+|<sub>/rest/logical-interconnects</sub>                                                   | GET       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | POST      | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |
```

Comment:
are we supporting this either in C7000 or Synergy?

Reply:
Done